src/Features/CSharp/Portable/MetadataAsSource/CSharpMetadataAsSourceService.cs

```diff
@@ -55,7 +55,7 @@ protected override async Task<Document> AddAssemblyInfoRegionAsync(Document docu
             return document.WithSyntaxRoot(newRoot);
         }
 
-        protected override IEnumerable<IFormattingRule> GetFormattingRules(Document document)
+        protected override IEnumerable<CodeAnalysis.Formatting.Rules.AbstractFormattingRule> GetFormattingRules(Document document)
```

Comment:
>CodeAnalysis.Formatting.Rules [](start = 39, length = 29)

nit: move to usings, and same below

Reply:
Name conflict. Resolved in a follow-up that I'll push later.

Reply:
Resolved in 053db37c20aa779f6d9dc72a1cd0d2b24a665e79